CMP1903M A01 2223/Testing.cs

```diff
@@ -11,43 +11,44 @@ public class Testing
         public static void Test()
         {
             //This Displays the pack
+            Console.WriteLine("This is card pack: ");
             Pack Deck = new Pack();
             Pack.DisplayPack();
 
             //The next section displays different shuffles
 
-            //This displays the Fisher-Yates shuffle
-            //Console.WriteLine(" This is the Fisher-Yates Shuffle: ");
-            //Pack.shuffleCardPack(1);
-            //Pack.DisplayPack();
+            //This displays the Fisher-Yates shuffle by calling the required functions
+            Console.WriteLine(" This is the Fisher-Yates Shuffle: ");
+            Pack.shuffleCardPack(1);
+            Pack.DisplayPack();
 
-            //This displays the Riffle Shuffle
-            //Console.WriteLine("This is the Riffle Shuffle");
-            //Pack.shuffleCardPack(2);
-            //Pack.DisplayPack();
+            //This displays the Riffle Shuffle by calling the required functions
+            Console.WriteLine("This is the Riffle Shuffle: ");
+            Pack.shuffleCardPack(2);
+            Pack.DisplayPack();
 
 
-            //This is not Shuffled
-            //Console.WriteLine("This is the pack not shufffled: ");
-            //Pack.shuffleCardPack(3);
-            //Pack.DisplayPack();
+            //This is not Shuffled by calling the required functions
+            Console.WriteLine("This is the pack not shufffled: ");
+            Pack.shuffleCardPack(3);
+            Pack.DisplayPack();
 
             //This section displays dealing the cards
 
-            //This is dealing one card
-            //Console.WriteLine("This is dealing one card");
-            //Pack.Deal().Show();
-            //Console.WriteLine("This is the rest of the deck");
-            //Pack.DisplayPack(); 
+            //This is dealing one card by calling the required functions
+            Console.WriteLine("This is dealing one card: ");
```

Comment:
I suggest you add spacing between writing lines as the console can get all clumped up but using something like \n can space the different outputs done by testing to make it easier to read